legolas-mysql/src/main/java/legolas/mysql/infra/MySQLStarter.java

```diff
@@ -0,0 +1,89 @@
+package legolas.mysql.infra;
+
+import legolas.net.core.interfaces.Port;
+import legolas.net.core.interfaces.SocketType;
+import legolas.runtime.core.interfaces.ServiceId;
+import legolas.sql.interfaces.DatasourceFactory;
+import legolas.sql.interfaces.SQLExecutor;
+import legolas.sql.interfaces.SQLStarter;
+import legolas.sql.interfaces.TargetDatabase;
+import legolas.mysql.interfaces.MySQLEntry;
+import legolas.mysql.interfaces.MySQLServiceId;
+import legolas.starter.api.interfaces.StarterComponent;
+import org.testcontainers.containers.MySQLContainer;
+
+import javax.sql.DataSource;
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+@StarterComponent
+public class MySQLStarter extends SQLStarter<MySQLContainer> {
+  static final String DEFAULT_PASSWORD = "my-secret-pw";
```

Comment:
I need to do like that because of mysql docker image documentation: *MYSQL_ROOT_PASSWORD
This variable is mandatory and specifies the password that will be set for the MySQL root superuser account. In the above example, it was set to my-secret-pw.* https://hub.docker.com/_/mysql

Used by test-container. Without this configuration, it didn't work.

Reply:
I've done it.